doc/source/programs/gdal_rasterize.rst

```diff
@@ -166,6 +166,8 @@ raster data is supported.
 
     Set output file size in pixels and lines. Note that :option:`-ts` cannot be used with
     :option:`-tr`
+    If one of the two values is set to 0, it will be computed from the other value in order to
+    preserve the aspect ratio.
```

Comment:
Is it so complicated? When the result would be something else than ResX=ResY?

Reply:
I don't follow: if it's a matter of wording can you suggest a better sentence?